modules/cockroachdb/options.go

```diff
@@ -67,3 +68,23 @@ func WithTLS(cfg *TLSConfig) Option {
 		o.TLS = cfg
 	}
 }
+
+// ClusterDefaults are the settings recommended by Cockroach Labs for testing clusters.
```

Comment:
suggestion: lets mention how this can be used in combination with `WithStatements`